lib/foreman_maintain/runner.rb

```diff
@@ -38,5 +43,23 @@ def ask_about_offered_steps(steps)
         @reporter.on_next_steps(self, steps)
       end
     end
+
+    def mock_skip_class(step)
+      Class.new(step.class.superclass) do
```

Comment:
Actually, this makes me ask what would be use case of using 'skipped' status?
https://github.com/iNecas/foreman_maintain/blob/master/lib/foreman_maintain/reporter/cli_reporter.rb#L204